notifications.yaml

```diff
@@ -0,0 +1,220 @@
+        type: "string"
+        format: "uuid"
+    get: 
+      operationId: "listMessagesByUser"
+      summary: "List Messages by user"
+      tags: 
+        - "Messages"
+      description: "Retrive all messages for this user. \n\nMessages will be returned in inverse delivery order (from last to first)"
+      responses: 
+        200: 
+          description: "Found."
+          schema: 
+            $ref: "#/definitions/MessageCollection"
+  /Messages: 
+    post: 
+      operationId: "sendMessage"
+      summary: "Send Message"
+      tags: 
+        - "Messages"
+      description: "Submit a message for delivery."
+      parameters: 
+        - 
+          name: "message"
+          in: "body"
+          schema: 
+            $ref: "#/definitions/MessageDefinition"
+      responses: 
+        202: 
+          description: "Message accepted for delivery."
+          schema: 
+            allOf: 
+              - 
+                properties: 
+                  id: 
+                    type: "string"
+                    format: "uuid"
+                  status: 
+                    $ref: "#/definitions/MessageStatus"
+                required: 
+                  - "id"
+                  - "status"
+            description: ""
+            type: "object"
+          headers: 
+            Location: 
+              type: "string"
+              description: "Location (URL) of accepted message resource.\nSend a GET request to this URL to retrieve the message status and details."
+definitions: 
+  ProblemJson: 
+    title: "Problem Type"
+    type: "object"
+    properties: 
+      type: 
+        type: "string"
+        format: "uri"
+        description: "An absolute URI that identifies the problem type.  When dereferenced,\nit SHOULD provide human-readable documentation for the problem type\n(e.g., using HTML).\n"
+        default: "about:blank"
+        example: "https://example.com/problem/constraint-violation"
+      title: 
+        type: "string"
+        description: "A short, summary of the problem type. Written in english and readable\nfor engineers (usually not suited for non technical stakeholders and\nnot localized); example: Service Unavailable\n"
+      status: 
+        type: "integer"
+        format: "int32"
+        description: "The HTTP status code generated by the origin server for this occurrence\nof the problem.\n"
+        minimum: 100
+        maximum: 600
+        exclusiveMaximum: true
+        example: 503
+      detail: 
+        type: "string"
+        description: "A human readable explanation specific to this occurrence of the\nproblem.\n"
+        example: "Connection to database timed out"
+      instance: 
+        type: "string"
+        format: "uri"
+        description: "An absolute URI that identifies the specific occurrence of the problem.\nIt may or may not yield further information if dereferenced."
+  MessageContent: 
+    title: "MessageContent"
+    type: "object"
+    properties: 
```

Comment:
right but the email *needs* this structure (if you want to send some formatting), otherwise you're stuck with plain text and/or messages without an object (or content).

if you stick to only one field (body_short) is that the content of the email or the object or both ?